UITests/BaseTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FlaUI.UIA3;
+using FlaUI.Core;
+using System;
+using System.Threading.Tasks;
+using EnvDTE;
+using AutomationWindow = FlaUI.Core.AutomationElements.Window;
+
+namespace UITests
+{
+   [TestClass]
+   public class BaseTest
+   {
+       protected static UIA3Automation _automation;
+       protected static Application _app;
+       protected static AutomationWindow _mainWindow;
+       protected static DTE _dte;
+
+       [ClassInitialize]
+       public static void ClassInitialize(TestContext context)
```

Comment:
can this method throw any exception during initialize process?
if yes, add try-catch to this logic